modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTCollectionTemplateLocalServiceImpl.java

```diff
@@ -92,6 +93,20 @@ public CTCollectionTemplate deleteCTCollectionTemplate(
 			CTCollectionTemplate ctCollectionTemplate)
 		throws PortalException {
 
+		long ctDefaultCTCollectionTemplateId =
+			_ctSettingsConfigurationHelper.getDefaultCTCollectionTemplateId(
+				ctCollectionTemplate.getCompanyId());
+
+		if (ctDefaultCTCollectionTemplateId ==
+				ctCollectionTemplate.getCtCollectionTemplateId()) {
```

Comment:
We can use `_ctSettingsConfigurationHelper.isDefaultCTCollectionTemplate` to check for default template.